spec/rack/test/uploaded_file_spec.rb

```diff
@@ -41,7 +41,7 @@ def test_file_path
         require 'java'
         java_import 'java.lang.System'
 
-        20.times do |i|
+        50.times do |i|
```

Comment:
Ah, the commit message makes sense. Could you cherry-pick this one out to a separate branch & PR? I'll approve it immediately then, so we can keep the history more reasonable.